pkg/shell/superuser.jsx

```diff
@@ -108,7 +110,12 @@ const UnlockDialog = ({ proxy, host }) => {
 
     function init() {
         return proxy.Stop().always(() => {
-            start("sudo");
+            if (proxy.Bridges.length == 1)
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-17536-20220708-114053-02201755-fedora-36-devel//Coverage/lcov/github-pr.diff.gcov.html)